ibethel/shorty
# Admin namespace 48

Resolves Ticket https://bethel-media.codebasehq.com/projects/dev-ops/tickets/48

WARNING: Merge in https://github.com/ibethel/shorty/pull/1 BEFORE you merge this!

Testing Instructions:
1. Visit the normal root url.
2. You should be immediately redirected to bethel.global
3. add "/admin" to the normal root url.
4. You should be brought to the normal sign in page.
5. Sign in.
6. You should be on the main shorts index.
7. Navigate around.
8. It should all work.
